Examples/Modules/qed/schwinger/analysis_schwinger.py

```diff
@@ -38,18 +38,27 @@
 # Find which test we are doing
 test_number = re.search( 'qed_schwinger([1234])', filename ).group(1)
 if test_number == '1':
+ # First Schwinger test with "weak" EM field. No pair should be created.
     Ex_test = 1.e16
     Bx_test = 16792888.570516706
     By_test = 5256650.141557486
     Bz_test = 18363530.799561853
 elif test_number == '2':
+# Second Schwinger test with stronger EM field. Many pairs are created and a Gaussian
+# distribution is used to get the weights of the particles. This is the most sensitive test
+# because the relative std is extremely low.
     Ex_test = 1.e18
     Bx_test = 1679288857.0516706
     By_test = 525665014.1557486
     Bz_test = 1836353079.9561853
 elif test_number == '3':
+# Third Schwinger test with intermediate electric field such that average created pair per cell
+# is 1. A Poisson distribution is used to obtain the weights of the particles.
     Ey_test = 1.0321239524474501e+17
 elif test_number == '4':
+# Fourth Schwinger test with extremely strong EM field but with E and B perpendicular and nearly
+# equal so that the pair production rate is fairly low. A Gaussian distribution is used in this
+# case.
     Ez_test = 2.5e+20
     By_test = 833910154604.3563
```

Comment:
```suggestion
    # First Schwinger test with "weak" EM field. No pair should be created.
    Ex_test = 1.e16
    Bx_test = 16792888.570516706
    By_test = 5256650.141557486
    Bz_test = 18363530.799561853
elif test_number == '2':
    # Second Schwinger test with stronger EM field. Many pairs are created and a Gaussian
    # distribution is used to get the weights of the particles. This is the most sensitive test
    # because the relative std is extremely low.
    Ex_test = 1.e18
    Bx_test = 1679288857.0516706
    By_test = 525665014.1557486
    Bz_test = 1836353079.9561853
elif test_number == '3':
    # Third Schwinger test with intermediate electric field such that average created pair per cell
    # is 1. A Poisson distribution is used to obtain the weights of the particles.
    Ey_test = 1.0321239524474501e+17
elif test_number == '4':
    # Fourth Schwinger test with extremely strong EM field but with E and B perpendicular and nearly
    # equal so that the pair production rate is fairly low. A Gaussian distribution is used in this
    # case.
    Ez_test = 2.5e+20
    By_test = 833910154604.3563
```